plugin-maven/src/test/java/com/diffplug/spotless/maven/incremental/UpToDateCheckingTest.java

```diff
@@ -56,6 +56,51 @@ void enableUpToDateChecking() throws Exception {
 		assertFormatted(files);
 	}
 
+	@Test
+	void enableUpToDateCheckingWithPluginDependencies() throws Exception {
+		writePomWithPluginManagementAndDependency();
+
+		List<File> files = writeUnformattedFiles(1);
+		String output = runSpotlessApply();
+
+		assertThat(output).contains("Up-to-date checking enabled");
+		assertFormatted(files);
+	}
+
+	@Test
+	void enableUpToDateCheckingWithPluginDependenciesMaven3_6_3() throws Exception {
+		writePomWithPluginManagementAndDependency();
+
+		setFile(".mvn/wrapper/maven-wrapper.properties").toContent("distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.6.3/apache-maven-3.6.3-bin.zip\n");
```

Comment:
Will this override the Maven version for all subsequent tests?

Reply:
No, each test gets its own directory.